tests/tox.ini

```diff
@@ -0,0 +1,4 @@
+; ignore all pydocstyle errors in this folder
+
+[pydocstyle]
+add_ignore = D
```

Comment:
Ignoring folders can't be done in the main tox.ini? I guess thats why we had this file match workaround?

Reply:
Not that I know of, see [here](http://www.pydocstyle.org/en/latest/usage.html#available-options)

Reply:
But I think this `tox.ini` in a subfolder has its advantages too

Reply:
all right